ios/native/NativeProxy.mm

```diff
@@ -363,6 +364,21 @@ static CFTimeInterval calculateTimestampWithSlowAnimations(CFTimeInterval curren
     }
     return nil;
   }];
+
+  [animationsManager setComputeSharedTransitionProgressAnimationForTagBlock:^(
+                         int viewTag, double progress, NSDictionary *snapshotValues) {
+    if (auto reaModule = weakModule.lock()) {
+      if (auto runtime = wrt.lock()) {
+        jsi::Runtime &rt = *runtime;
+        jsi::Value snapshotJSValues = convertNSDictionaryToJSIObject(rt, snapshotValues);
```

Comment:
I don't understand why snapshot is passed down from JS. Was it handled the same way before with regular shared element transitions?

Reply:
I got rid of this additional conversion 👍